src/signaling/initiator.ts

```diff
@@ -303,6 +303,27 @@ export class InitiatorSignaling extends Signaling {
         }
     }
 
+    /**
+     * Drop a new responder after a handshake with one responder has already
+     * completed.
+     *
+     * Note: This deviates from the intention of the specification to allow
+     *       for more than one connection towards a responder over the same
+     *       WebSocket connection.
+     */
+    protected onUnhandledSignalingServerMessage(msg: saltyrtc.Message): void {
+        if (msg.type === 'new-responder') {
+            try {
+                this.log.debug(this.logTag, 'Received new-responder');
```

Comment:
If you want to be consistent with log messages in `onSignalingMessage`, append " message" to the log. Same for the responder.